core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -61,6 +61,7 @@ const disableInteractiveOption = {
   vertexMove: false,
   vertexAdd: false,
   vertexRemove: false,
+  elementMove: false,
```

Comment:
just a clarification question, does `element` include all kinds of elements (`link`, `label`, ... etc)? It will be good to leave some comments here since you are introducing a new option, and it's better to be explicitly clear about its definition.

Reply:
Yes. This change is temporary, I'll introduce another level of 'disable'.

Reply:
ok. better to keep comments about temporary changes in code, even better to leave todos in comments to explain the future intension, so that it won't cause confusion.

Reply:
Comments added.